libs/pubsub/pubsub.go

```diff
@@ -404,7 +406,7 @@ func (state *state) send(msg interface{}, tags map[string]string) {
 					// don't block on buffered channels
 					select {
 					case subscription.out <- Message{msg, tags}:
-					default:
+					case <-time.After(2 * time.Second): //give some time for the subscriber to handle
```

Comment:
Why was this necessary?

I don't think we want this, especially not with a hard coded time. We just refactored this module to not block here.